tests/spec/corpus.rs

```diff
@@ -247,6 +250,40 @@ fn run_test(test: TestFile) {
             );
         }
     }
+
+    for decode_error in test.decode_errors {
+        // No meaningful definition of "byte count" for an arbitrary reader.
```

Comment:
Perhaps I'm misunderstanding this test, but I don't see why there can't be other data after the section identified as being the document.

Reply:
Since this is in the top-level file, I think the idea is that if that the user tries to decode a single document and then provides more bytes than the document indicates, it should fail. I'm not really sure how we could represent that in our tests though, since our readers are lazy.

Reply:
Is it okay to just continuing to skip this test then?

Reply:
I'm fine with skipping it